draft-ietf-rats-corim.md

```diff
@@ -809,7 +815,8 @@ The following describes each member of the `measurement-map`:
 
 ###### Measurement Keys {#sec-comid-mkey}
 
-The types defined for a measurement identifier are OID, UUID or uint.
+A measurement-associated (sub-)environment is at the granularity of a single element, such as the second enumerated network interface card.
+The types of this measured element identifier are OID, UUID or uint.
```

Comment:
Since there is a pending discussion around whether or not we want to key mkey. We should hold off on making changes to mkey related content until that discussion is resolved.